### ISSUE TITLE
Feature: Integration with Ethereum / IPFS

### ISSUE THREAD

**Issue by juanfranblanco** [ConsenSys/acid-reflux#1]
This is a very good start up example for an application, it will be great to see it as and end to end example (web3, ipfs), using truffle, embark, etc.

**Comment by jeffscottward** [comment]
@juanfranblanco Absolutely. Would love to have both EthereumJS and the Javscript version of IPFS embedded right in there rather than having to ping an outside service. Except they don't quite exist yet :-\.

Planning on making a Webpack loader in the works for compiling solidity and generating javascript methods using Pudding.whisk (a tool inside Truffle). I'm sure others may do the same for Gulp/Grunt/etc.

Embark and Truffle should be merging I believe but the idea is to stay very low level and bring in the small bits rather than rely on truffle or embark outright because otherwise you need to rely on outside updates to the aggregated frameworks rather than small bits you use.

I'm cleaning this up a little more for now and adding the above functionality.

**Comment by juanfranblanco** [comment]
That is fantastic many thanks!
